app/queries/project_filter_query.rb

```diff
@@ -19,7 +19,7 @@ def items
     items = @project
       .content_items
       .with_valid_ids
-      .order(updated_at: :desc)
+      .order(id: :asc)
```

Comment:
I think we wanted to change the order of the to-do view only, so we could change the order in the case statement below instead.

Reply:
That sounds fine, I've updated the commit.